### PR TITLE
feat: unidock_pipeline add support for score_only and local_only; doc: update  unidock perf png

### PR DESCRIPTION
**Pull request by dp-yuanyn**

### CODE REVIEW

**Review by caic99 — APPROVED** [review]

File: unidock/assets/1.1_docking_box.png
Comment (zhengh96) [inline unidock/assets/1.1_docking_box.png:1]: I think this update should be a independent PR.
